Fix ru_locale strings and arrays

mesnevi opened the pull request:
1) Fix incorrect, awkward and machine-translated strings

2) Regroup strings (they were ordered alphabetically;  it was pretty
confusing: related values, originally grouped by type ['settings_'
or 'action_' etc.) were scattered all around the xml) to fit the
original EN XML file

3) Fix missing, incorrect, awkward and machined-translated arrays

4) Add missing strings (it seems that many strings were added in
EN locale, but were missing in Russian one) and arrays

5) Perhaps it was not worth translating commented out <explore-items>
portion. I thought it was probably tagging work in progress, so if
it's not needed, I can remove it.

Signed-off-by: Lesya Novaselskaya <shams@airpost.net>

Commit:
Fix ru_locale strings and arrays

1) Fix incorrect, awkward and machine-translated strings

2) Regroup strings (they were ordered alphabetically;  it was pretty
confusing: related values, originally grouped by type ['settings_'
or 'action_' etc.) were scattered all around the xml) to fit the
original EN XML file

3) Fix missing, incorrect, awkward and machined-translated arrays

4) Add missing strings (it seems that many strings were added in
EN locale, but were missing in Russian one) and arrays

5) Perhaps it was not worth translating commented out <explore-items>
portion. I thought it was probably tagging work in progress, so if
it's not needed, I can remove it.

6) Fix date format

Signed-off-by: Lesya Novaselskaya <shams@airpost.net>

relan commented:
I'm native Russian speaker and I can confirm that this pull request greatly improves translation.

@WangDaYeeeeee, could you please merge it?

mesnevi commented:
This patch greatly improves the app usage for Russian speaking folks. @WangDaYeeeeee, could you very please merge it? Current translation seems to be partly machine-translated (thus of mediocre quality).

Commit:
Fix ru locale author name

RU translation is not simply fixed up, it was rather reworked.
Hence fix translator name.

Signed-off-by: Lesya Novaselskaya <shams@airpost.net>
